packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -592,32 +602,20 @@ export function buildWebpackBrowser(
               }
 
               let bundleInfoText = '';
-              const processedNames = new Set<string>();
-              for (const result of processResults) {
-                processedNames.add(result.name);
-
-                const chunk =
-                  webpackStats &&
-                  webpackStats.chunks &&
-                  webpackStats.chunks.find(c => result.name === c.id.toString());
-                if (result.original) {
-                  bundleInfoText +=
-                    '\n' + generateBundleInfoStats(result.name, result.original, chunk);
-                }
-                if (result.downlevel) {
-                  bundleInfoText +=
-                    '\n' + generateBundleInfoStats(result.name, result.downlevel, chunk);
-                }
-              }
-
-              if (webpackStats && webpackStats.chunks) {
-                for (const chunk of webpackStats.chunks) {
-                  if (processedNames.has(chunk.id.toString())) {
-                    continue;
+              for (const chunk of webpackStats.chunks) {
```

Comment:
Can you keep this part as-is?  There are potential processed files that are not Webpack chunks. Web worker files are one such case.

Reply:
Updated this to run `generateBundleStats()` on all chunks which don't have an associated `processResult`, which I think it was this was trying to do.

I made a simple project with a web worker, but I don't see that show up in `chunks` or `processResults` and it doesn't not display bundle size information even when built with the CLI from master. I'm not sure exactly what this is supposed to do in that case, so I can verify that my changes don't alter than behavior.

Reply:
They won't show until #16290 is merged.